fediplay/mastodon.py

```diff
@@ -42,13 +48,20 @@ def login(client, grant_code):
     client.log_in(code=grant_code, to_file='usercred.secret')
     umask(old_umask)
 
-def stream(api_base_url):
+def stream(api_base_url, users):
     '''Stream statuses and add them to a queue.'''
 
     client = Mastodon(client_id='clientcred.secret', access_token='usercred.secret',
                       api_base_url=api_base_url)
     listener = StreamListener(Queue())
     print('==> Streaming from', api_base_url)
+    clientInstanceTmp = client.instance().uri
```

Comment:
The great news is that we actually use `instance` here now in the 2.0 codebase, so we don't have to do this bit anymore. 👍